learning_assistant/models.py

```diff
@@ -1,3 +1,39 @@
 """
 Database models for learning_assistant.
 """
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class CoursePrompt(TimeStampedModel):
+    """
+    This model represents a mapping between a particular course ID and a text prompt associated with the course ID.
+
+    .. no_pii: This model has no PII.
+    """
+
+    # course ID with which the text prompt is associated with
+    course_id = models.CharField(max_length=255, db_index=True, unique=True)
+
+    # text prompt, that may contain course related information
+    prompt = models.TextField(blank=True)
+
+    class Meta:
```

Comment:
Is there a reason you included the Meta class? I believe that the values you set for `db_table` and `verbose_name` should be the default values that Django will compute and user under-the-hood.

Reply:
I had seen it included for most other models in repos that we own - I can remove it though!

Reply:
Hm. You could try removing the Meta class, recreating the migration, running the migration, and see if the effect is the same in the database? I'm usually in favor of not having the code if the framework will do it the same way anyway.

Could it be that those other models have app names that we don't want in the table name? For example, I see `edx_proctoring` models have a Meta class with `db_table`, but I think that's to avoid a table name of `edxproctoring_proctoredexam`, so we set it to `proctoring_proctoredexam`. Maybe we just copied it from `edx-proctoring`?

Reply:
Yeah, it's probably something we've copied over time. It looks fine without the meta class 👍